examples/trixi_imex_ars.jl

```diff
@@ -0,0 +1,37 @@
+# # Using an implicit-explicit (IMEX) Runge-Kutta solver based on Ariadne with Trixi.jl
+
+using Trixi
+using Theseus
+using CairoMakie
+
+# Notes:
+# You must disable both Polyester and LoopVectorization for Enzyme to be able to differentiate Trixi.jl.
+#
+# LocalPreferences.jl
+# ```toml
+# [Trixi]
+# loop_vectorization = false
+# backend = "static"
+# ```
+
+@assert Trixi._PREFERENCE_THREADING !== :polyester
+@assert !Trixi._PREFERENCE_LOOPVECTORIZATION
+
+# First call to load callbacks
+trixi_include(joinpath(examples_dir(), "tree_2d_dgsem", "elixir_advection_diffusion.jl"), sol = nothing);
```

Comment:
```suggestion
trixi_include(@__MODULE__, joinpath(examples_dir(), "tree_2d_dgsem", "elixir_advection_diffusion.jl"), sol = nothing);
```

Documenter.jl runs this in a separate module, not in `Main` (the default used by `trim_include`).